.github/workflows/build.yml

```diff
@@ -36,5 +36,7 @@ jobs:
         run: echo ${{ env.ACTIVE_STASH_TEST_COLLECTION_PREFIX }}
       - name: "Clear out database"
         run: psql -d ${{ env.PGDATABASE }} -c 'DROP SCHEMA public CASCADE; CREATE SCHEMA public;'
+      - name: "Install default uri gem version"
+        run: gem install --default -v0.11.0 uri
```

Comment:
Do we need to update our installation docs so that users don't see the same issue that this PR fixes in CI?

Reply:
yep possibly. I'm testing this out locally in a Rails 7 app with Ruby 2.7.4 and not seeing the same error we see in CI. I'm trying to working out why.

Reply:
I can't replicate this uri error in an Active Stash Rails 7 app with Ruby 2.7.4.

Running `gem dependency`, i'm not seeing 2 versions returned, only 0.11.0.

<img width="321" alt="Screen Shot 2022-06-21 at 9 17 56 am" src="https://user-images.githubusercontent.com/26052576/174687961-e69b02fa-4a7e-4422-91b0-947c4110eb10.png">